examples/ws/soap/StoreExample/StoreServer/WebContent/WEB-INF/MyStoreServerService/DefaultNamespace/deploy.wsdd.bak

```diff
@@ -0,0 +1,32 @@
+<!-- Use this file to deploy some handlers/chains and services      -->
```

Comment:
Why do we need a `.bak` file?

Reply:
I think that file is automatically generated, no idea what it is.

Reply:
Then we should not add it to the repo.